playbooks/cnf/deploy-run-downstream-tests-script.yaml

```diff
@@ -0,0 +1,134 @@
+---
+## Disclaimer:
+# This playbook is not officially supported and comes with no guarantees.
+# Use it at your own risk. Ensure you test thoroughly in your environment
+# before deploying to production.
+
+# Ansible Playbook: Set up downstream cnf-gotests project
+# -------------------------------------------------
+# This playbook clones cnf-gotests project and deploys all external dependencies
+# based on given parameters. The playbook uses following algorithm:
+
+# -----------------------------------------------------------------------------
+# Playbook Name: Set up cnf-gotests project
+# Description  : This playbook prepares the environment on the bastion host
+#                to run CNF downstream tests. It ensures dependencies are
+#                installed, clones the appropriate test repository, and sets
+#                up the test execution script.
+#
+# Prerequisites:
+#   - Variables `metallb_vlans` and `switch_interfaces`,`kubeconfig` must be defined
+#   - The `kubeconfig` must be available to authenticate with the cluster
+#   - Go must be installed on the system where this playbook is run
+#
+# Key Tasks:
+#   - Gather cluster version from OpenShift
+#   - Clone the downstream test repository
+#   - Set up Ginkgo testing tool
+#   - Generate test execution script from a Jinja2 template using given parameters
+
+# Usage:
+# Run this playbook using Ansible:
+# ansible-playbook ./playbooks/cnf/deploy-run-downstream-tests-script.yaml -i ./inventories/cnf/switch-config.yaml \
```

Comment:
small nit - we have two different files with the same name `switch-config.yaml`

1. is an inventory 
2. is a playbook

I think we should consider changing this.
not a blocker 

/lgtm